.github/workflows/enos-ci-cleanup.yml

```diff
@@ -0,0 +1,92 @@
+          aws-region: us-east-1
+          role-to-assume: ${{ secrets.AWS_ROLE_ARN_CI }}
+          role-skip-session-tagging: true
+          role-duration-seconds: 3600
+      - name: Get all regions
+        id: setup
+        run: |
+          echo "regions=$(aws ec2 describe-regions --region us-east-1 --output json --query 'Regions[].RegionName' | tr -d '\n ')" >> $GITHUB_OUTPUT
+
+  aws-nuke:
+    needs: setup
+    runs-on: ubuntu-latest
+    container:
+      image: rebuy/aws-nuke
+      options:
+         --user root
+         -t
+      env:
+        AWS_ACCESS_KEY_ID: ${{ env.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ env.AWS_SECRET_ACCESS_KEY }}
+        TIME_LIMIT: "72h"
+    timeout-minutes: 60
+    steps:
+      - name: Configure AWS credentials
+        id: aws-configure
+        uses: aws-actions/configure-aws-credentials@v1-node16
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_CI }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_CI }}
+          aws-region: us-east-1
+          role-to-assume: ${{ secrets.AWS_ROLE_ARN_CI }}
+          role-skip-session-tagging: true
+          role-duration-seconds: 3600
+          mask-aws-account-id: false
+      - uses: actions/checkout@v3
+      - name: Configure
+        run: |
+          cp enos/ci/aws-nuke.yml .
+          sed -i "s/ACCOUNT_NUM/${{ steps.aws-configure.outputs.aws-account-id }}/g" aws-nuke.yml
+          sed -i "s/TIME_LIMIT/${TIME_LIMIT}/g" aws-nuke.yml
+      # We don't care if cleanup succeeds or fails, because dependencies be dependenceies,
+      # we'll fail on actually actionable things in the quota steep afterwards.
+      - name: Clean up abandoned resources
+        # Filter STDERR because it's super noisy about things we don't have access to
+        # this without --no-dry-run will just print what would change (for testing)
+        run: |
+          aws-nuke -c aws-nuke.yml -q --force 2>/tmp/aws-nuke-error.log || true
+
+  check-quotas:
+    needs: [ setup, aws-nuke ]
+    runs-on: ubuntu-latest
+    container:
+      image: jantman/awslimitchecker
+      env:
+        AWS_ACCESS_KEY_ID: ${{ env.AWS_ACCESS_KEY_ID_CI }}
+        AWS_SECRET_ACCESS_KEY: ${{ env.AWS_SECRET_ACCESS_KEY_CI }}
+    strategy:
+      matrix:
+        region: ${{ fromJSON(needs.setup.outputs.regions) }}
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1-node16
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_CI }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_CI }}
+          aws-region: us-east-1
+          role-to-assume: ${{ secrets.AWS_ROLE_ARN_CI }}
+          role-skip-session-tagging: true
+          role-duration-seconds: 3600
+      # Currently just checking VPC limits across all region, can add more checks here in future
+      - name: Check AWS Quotas
```

Comment:
I guess for now, we just need to remember to check the result of this workflow. Would be nice to get a notification in slack.